packages/chain-adapters/src/api.ts

```diff
@@ -81,6 +81,8 @@ export type ChainAdapter<T extends ChainId> = {
     onError?: (err: SubscribeError) => void,
   ): Promise<void>
 
+  parseTx(tx: unknown, pubkey: string): Promise<Transaction>
```

Comment:
@kaladinlight FYI ^^ not private anymore so we can access this from clients as needed